software/binpastes.yml

```
@@ -0,0 +1,11 @@
+name: BinPastes
+website_url: https://github.com/querwurzel/BinPastes
+source_code_url: https://github.com/querwurzel/BinPastes
+description: "Minimal pastebin supporting client-side encryption, fulltext search, one-time messages. Intended for one to few users looking for a simple pastebin deployment."
+licenses:
+  - Apache-2.0
+platforms:
+  - Java
+tags:
+  - Pastebins
+depends_3rdparty: true
```

```suggestion
depends_3rdparty: true
```

Why does this apply?

---

Depends on what counts as 3rd party. You need a runtime environment (JDK) and the software uses libraries.

---

I see. It does not apply then, please remove this line.

> https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md
> whether the software depends on a third-party service outside the user's control

---

Allright, set it to false